Packs/CTIX/Integrations/CTIX/CTIX.yml

```diff
@@ -31,6 +31,25 @@ configuration:
   name: proxy
   type: 8
   required: false
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: C - Fairly reliable
+  display: Source Reliability
+  name: integrationReliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: false
+  type: 15
+- display: ""
+  name: feedExpirationPolicy
```

Comment:
arent we missing this here? 
options:
  - never
  - interval
  - indicatorType
  - suddenDeath

Reply:
checking with Dror

Reply:
he said the way i did is good enough, @Ni-Knight FYI

Reply:
It should be enough, since up to today it had none.